akka-actor-typed-tests/src/test/scala/docs/akka/typed/IntroSpec.scala

```diff
@@ -24,16 +22,51 @@ object IntroSpec {
   //#hello-world-actor
   object HelloWorld {
     final case class Greet(whom: String, replyTo: ActorRef[Greeted])
-    final case class Greeted(whom: String)
+    final case class Greeted(whom: String, from: ActorRef[Greet])
 
-    val greeter = Behaviors.immutable[Greet] { (_, msg) ⇒
+    val greeter = Behaviors.immutable[Greet] { (ctx, msg) ⇒
```

Comment:
Other behavior definitions have a type ascription on val/def instead of passing a type parameter. We should keep it consistent and add a type ascription here for the lowest cognitive noise.

Reply:
good point, fixed